indico/util/string.py

```diff
@@ -129,20 +129,28 @@ def strict_unicode(value):
     return unicode(value)
 
 
-def slugify(value, lower=True):
+def slugify(value, lower=True, maxlen=0):
```

Comment:
True, even though a "length" is obviously a number. My thought was that a length of 0 and no length (None) are different things, even though it's common sense that you never want a zero-length slug.